CONTRIBUTORS.md

```diff
@@ -0,0 +1,15 @@
+# Contributors List
+
+This is the list of Ed-Fi-ODS-Implementation's significant contributors.
+
+This does not necessarily list everyone who has contributed code, especially
+since many employees of one corporation may be contributing. To see the full
+list of contributors, see the revision history in source control or the [GitHub
+contributors
+graph](https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-Standard/graphs/contributors).
+New contributors should add their name and e-mail address or link to GitHub
+profile to this file with their first pull request.
+
+Ed-Fi Alliance\
+Mr. Ed <talkinghorse@ed-fi.org>\
+[Gumby](https://github.com/this-is-a-fake-profile)
```

Comment:
I think yoy can remove from line 13 to 15 since this contributor is just an example

Reply:
Fixed